packages/ds-healthcare-gov/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cmsgov/ds-healthcare-gov",
-  "version": "6.2.0-beta.1",
+  "version": "6.3.0-beta.1",
```

Comment:
This jumped from beta to beta because it was migrated prior to the full `6.2.0` release and never had its version number updated